.fixtures.yml

```diff
@@ -0,0 +1,8 @@
+fixtures:
+    symlinks:
+      tea:  "#{source_dir}"
+      test: "#{source_dir}/spec/fixtures/test"
+    repositories:
+      stdlib:
```

Comment:
Yes, stdlib - it can probably be removed, there's no dependency on it that I can see.